### ISSUE TITLE
Warzone spawns

### ISSUE THREAD

**Issue by mDerinho** [SmashMC-Development/Suggestions#856]
Shrine legendaries should be removed in my opinion.  Since legendaries like Dialga, Palkia, Giratina, Arceus, Zapdos, Moltres spawn in WZ, the use of shrines, especially when its pay to get shrines. It makes the shrines kinda pointless/a bad way to get the legends. It also means that the value of these legendaries is very low, when they should be higher. It is too easy to get them in wz compared to the way you would actually obtain them. Red chain for example requires the lake trio and is one of the most valuable items which is now useless, because of the amount of dialga palkia giratina spawning.

**Comment by DinglePringleUwU** [comment]
Or maybe make those spawns more rare in wz. Also meltans/melmetals, which could let to more people mining and selling ores :D

**Comment by mDerinho** [comment]
Exactly the lines im thinking along, there's alot of pokemon that dont spawn naturally and i belive they shoudl be more rare than pokemon that naturally would spawn in the wild{overworld} There is alot of legendaries that dont like lugia,ho-oh, galar bird trio, normal bird trio, celebi, phione, urshifu(only kubfu spawning in WZ would be better imo) and others legends

**Comment by HobnobberJunior** [comment]
I totally agree with this. I does ruin the point of having shrines and getting meltan the regular way.

**Comment by ajdawg48** [comment]
I don’t really agree with this because then you can just make the argument to get every legend the “normal way”. It would be taking out a big part of warzone that brings people to it and want to keep going in it as it would be lowering the reward possibilities. I think they should all stay in warzone as it will keep warzone more active and players will keep going to it. If players were limited to getting certain legends the normal way would be defeating big part of warzone.

**Comment by HobnobberJunior** [comment]
This might be true, however it doesnt take away the fact that shrines are more or less useless now, which is a shame. On the other hand I also think that there are enough legends in the game, so that if u remove the shrine legends, wz would still have plenty of them.

**Comment by Mikirae** [comment]
I see both sides of this, so it's actually a pretty tough suggestion for me. Lol
Articuno, Zapdos, Moltres, Meltan, Melmetal, Urshifu, Lugia, Ho-Oh, Dialga, Palkia, Giratina, Arceus, and Celebi would all be removed.
I think a compromise of this would be making these mons more rare compared to other mons, but I look forward to hearing any further opinions on this before I bring it up to the rest of management :)

**Comment by HobnobberJunior** [comment]
This would maybe be a solution that could work. As of right now the price of one of the trio birds is less than the cost of a shrine which makes them a wee bit pointless :)

**Comment by Mikirae** [comment]
I asked for opinions and changed them the other day though :(

**Comment by HobnobberJunior** [comment]
Yes and I agree that the prices of the shrines now are correct. However since we have had SO many trio spawns, people have way too many of them and selling them for cheaper than what the shrine costs. I do not think there is anything wrong with the prices of the shrines just the sheer amount of legendary shrine spawns.
